stretch_deep_perception/package.xml

```diff
@@ -1,91 +1,58 @@
 <?xml version="1.0"?>
```

Comment:
can we coalesce the various <*_depends> into just the <depends> tag.

Reply:
Updated all tags to <depend> in PR #28.